glow/rate_limiter_test.go

```diff
@@ -0,0 +1,84 @@
+package glow
+
+import (
+	"testing"
+	"time"
+)
+
+func TestRateLimiterSingle(t *testing.T) {
+	rl := NewRateLimiter(1, 50*time.Millisecond)
+
+	ticker := time.NewTicker(20 * time.Millisecond)
+	defer ticker.Stop()
+
+	count := 0
+	allowed := 0
+	for _ = range ticker.C {
+		if rl.Allow() {
+			allowed++
+		}
+		count++
+		if count == 4 {
+			break
+		}
+	}
+
+	t.Logf("allowed %v", allowed)
```

Comment:
Generally discouraged to use Logf. There's a few places in the test suite where I use it, but nobody knows how to read them and nobody knows when they are wrong or suspicious.

Best to remove this line entirely

Reply:
The main category of thing that I use `t.Log` for is to expose some piece of information which is useful to know but is also going to be changing all the time. And honestly, I should probably prefix all of the tests that log stuff like that with `TestInfo` so that they can easily be filtered for, and the log message itself should explain what the value is and why it's interesting to someone running tests.

Reply:
Fair enough, I noticed there were not too many logfs in there